### ISSUE TITLE
Mark urllib2 as dependency in pip

### ISSUE THREAD

**Issue by pmatos** [lukaszbanasiak/yahoo-finance#75]
Is it possible to mark urllib2 as a pip dependency?

```
$ pip install --upgrade yahoo-finance
Collecting yahoo-finance
  Downloading yahoo-finance-1.3.1.tar.gz
Requirement already up-to-date: pytz in ./env/lib/python3.5/site-packages (from yahoo-finance)
Requirement already up-to-date: simplejson in ./env/lib/python3.5/site-packages (from yahoo-finance)
Building wheels for collected packages: yahoo-finance
  Running setup.py bdist_wheel for yahoo-finance ... done
  Stored in directory: /home/pmatos/.cache/pip/wheels/3f/9c/83/957631502e76761e43527715be35bb1ab3771939b6d7ee60a4
Successfully built yahoo-finance
Installing collected packages: yahoo-finance
  Found existing installation: yahoo-finance 1.2.2
    Uninstalling yahoo-finance-1.2.2:
      Successfully uninstalled yahoo-finance-1.2.2
Successfully installed yahoo-finance-1.3.1
(env)  [0]  pmatos@andromeda.desktop:~/Projects/portman
(5) $ python
Python 3.5.1 (default, Sep 19 2016, 10:16:17) 
[GCC 6.1.1 20160621 (Red Hat 6.1.1-3)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from yahoo_finance import Share
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/pmatos/Projects/portman/env/lib/python3.5/site-packages/yahoo_finance/__init__.py", line 1, in <module>
    import yahoo_finance.yql
  File "/home/pmatos/Projects/portman/env/lib/python3.5/site-packages/yahoo_finance/yql.py", line 36, in <module>
    import urllib2
ImportError: No module named 'urllib2'
>>>
```

**Comment by lukaszbanasiak** [comment]
@pmatos working on that

**Comment by pmatos** [comment]
@lukaszbanasiak Wow, you're fast. Thanks.

**Comment by pmatos** [comment]
Apologies, pressed the wrong button (Close and comment).

**Comment by lukaszbanasiak** [comment]
@pmatos fixed in 1.3.2 https://github.com/lukaszbanasiak/yahoo-finance/releases/tag/1.3.2

**Comment by pmatos** [comment]
@lukaszbanasiak verified. Thanks.
